cmd/cdi-apiserver/apiserver.go

```diff
@@ -119,6 +121,15 @@ func main() {
 
 	snapClient := snapclient.NewForConfigOrDie(cfg)
 
+	cluster, err := cluster.New(cfg)
+	if err != nil {
+		klog.Fatalf("Unable to create controller runtime cluster: %v\n", errors.WithStack(err))
+	}
+
+	if err := cdiv1.AddToScheme(cluster.GetScheme()); err != nil {
```

Comment:
Aren't some schemes missing here? for example https://github.com/kubevirt/containerized-data-importer/blob/a663355f98938a4136ddabe60f599a8b77ef7fcb/cmd/cdi-controller/controller.go#L67 what am I missing

Reply:
Not needed yet, the client just gets cdi resources